### PR TITLE
Refer to W3C Anti-Trust Policy

### PR DESCRIPTION
**Pull request by seanturner**

### CODE REVIEW

**Comment by AramZS** [comment]
I think this is a good change to resolve some of the open questions posed by #18 that we can resolve separately and immediately. I approve and I think if we see no objections (we are basically just stating the existing policy as a W3C group that we would already have) we can consider this having consensus by end of week.

File: charter.html [hunk 1]
```diff
@@ -62,6 +62,7 @@
           <li><a href="#decisions">Decision Policy</a></li>
           <li><a href="#patentpolicy">Patent Policy</a></li>
           <li><a href="#licensing">Licensing</a></li>
+	  <li><a href="#anti-trust">Licensing</a></li>
```
Comment (martinthomson) [inline charter.html:65]: Indent looks off.  Have you considered using [tidy](https://github.com/htacg/tidy-html5)?

**Review by martinthomson — APPROVED** [review]
How many sections of this document will be reduced to "we'll follow established W3C practices for X"?  Is there any way to refactor all of these into a single statement that covers all of these?

**Comment by AramZS** [comment]
No objections.

File: charter.html [hunk 2]
```diff
@@ -62,6 +62,7 @@
           <li><a href="#decisions">Decision Policy</a></li>
           <li><a href="#patentpolicy">Patent Policy</a></li>
           <li><a href="#licensing">Licensing</a></li>
+	  <li><a href="#anti-trust">Anti-Trust</a></li>
```
Comment (seanturner) [inline charter.html:65]:
```suggestion
          <li><a href="#licensing">Licensing</a></li>
	  <li><a href="#anti-trust">Anti-Trust</a></li>
```

Comment (seanturner) [inline charter.html:65]:
```suggestion
          <li><a href="#licensing">Licensing</a></li>
	  <li><a href="#anti-trust">Anti-Trust</a></li>
```

**Pull request merged**